README.md

```diff
@@ -175,6 +175,28 @@ Volume snapshot APIs and client library are now in a separate sub-module: `githu
 
 Use the command `go get -u github.com/kubernetes-csi/external-snapshotter/client/v3@v3.0.0` to get the client library.
 
+### Setting Quota limits with Snapshot custom resources
+[`ResourceQuotas`](https://kubernetes.io/docs/concepts/policy/resource-quotas/) are namespaced objects that can be used to set limits on objects of a particular [`Group.Version.Kind`](https://book.kubebuilder.io/cronjob-tutorial/gvks.html). Before we set resource quota, make sure that snapshot CRDs are installed in the cluster. If not please follow [this guide](https://github.com/kubernetes-csi/csi-driver-host-path/blob/master/docs/deploy-1.17-and-later.md)
+```
```

Comment:
I'd just reference the Usage section of README as deploy-1.17-and-later.md will need an update as well.

Reply:
that makes sense. updated.

Reply:
Add a period at the end of the sentence.